runtime/lite/src/consumer.rs

```diff
@@ -19,48 +25,44 @@ use dotenv::dotenv;
 #[tokio::main]
 async fn main() {
     dotenv().ok();
-    let db = CoreStorage::new("runtime_db").unwrap();
+    let db = CoreStorage::new("runtime-db").unwrap();
     let logger = CoreLogger::new(Some("./ssb-consumer.log"));
     let state_manager = GlobalState::new(logger.clone());
 
     logger.info("starting consumer...");
 
     let context = Arc::new(Mutex::new(Context::new(logger.clone(), db, state_manager)));
 
-    let secret = std::env::var("CONSUMER_SECRET").unwrap_or_else(|_| {
-        let home_dir = dirs::home_dir().unwrap();
-        std::format!("{}/.ssb/secret", home_dir.to_string_lossy())
-    });
-    let port = std::env::var("CONSUMER_PORT").unwrap_or_else(|_| 8008.to_string());
+    let secret = std::env::var("CONSUMER_SECRET")
+        .unwrap_or_else(|_| DEFAULT_CONSUMER_SECRET_DIR.to_string());
+    let port = std::env::var("CONSUMER_PORT").unwrap_or_else(|_| DEFAULT_CONSUMER_PORT.to_string());
+    let ip = std::env::var("CONSUMER_IP").unwrap_or(DEFAULT_CONSUMER_IP.to_string());
     let mut file = async_std::fs::File::open(secret).await.unwrap();
     let key = read_patchwork_config(&mut file).await.unwrap();
 
     let ssb_context = context.clone();
 
     let (channel_invite_tx, channel_invite_rx) = std::sync::mpsc::channel::<String>();
-    let mut client = Client::new(Some(key), "0.0.0.0".to_string(), port)
-        .await
-        .unwrap();
+    let mut client = Client::new(Some(key), ip, port).await.unwrap();
     // Spawn the SSB feed listener task
     tokio::spawn(async move {
-        let logger = ssb_context.clone().lock().unwrap().get_logger().clone();
```

Comment:
Code are missing from here, please update with runtime dev branch

Reply:
please check the code at line number 54. I have moved the code.